### PR TITLE
Create e-learning solution page

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Create menu icon
- [x] Add to solution menu
- [x] Add to solution index
- [x] Add redirect from /e-learning

### CODE REVIEW

**Comment by michellusthof** [comment]
[Preview](https://deploy-preview-2151--defacto-nl.netlify.com/maak-zelf-e-learning/)

File: data/redirects_nl.yml [hunk 1]
```diff
@@ -16,3 +16,4 @@
 - /compliance-api /capp-compliance-api
 - /capp-lms /capp-learning
 - /hosting-security /saas
+- /e-learning /maak-zelf-e-learning
```
Comment (RonaldDijkstra) [inline data/redirects_nl.yml:19]: Please do this in another PR and remove the old page completely for dutch.

Comment (michellusthof) [reply data/redirects_nl.yml:19]: Will do. Thanks
